MSAL/test/automation/tests/MSALBaseUITest.m

```diff
@@ -386,9 +389,58 @@ - (void)closeAuthUIUsingWebViewType:(MSIDWebviewType)webViewType
     }
 }
 
-- (void)closeResultView
+- (void)cleanPipelines
+{
+    static NSArray *pipelinesPaths = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        pipelinesPaths = @[
+            [MSIDAutomationActionConstants requestPipelinePath],
+            [MSIDAutomationActionConstants resultPipelinePath],
+            [MSIDAutomationActionConstants logsPipelinePath]
+        ];
+
+    });
+    
+    for (NSString *path in pipelinesPaths)
+    {
+        if (![NSFileManager.defaultManager fileExistsAtPath:path]) continue;;
+        
+        // Delete file.
+        NSError *error;
+        BOOL fileRemoved = [NSFileManager.defaultManager removeItemAtPath:path error:&error];
+        XCTAssertNil(error);
+        XCTAssertTrue(fileRemoved);
+    }
+}
+
+- (void)closeResultPipeline
```

Comment:
Would this code fit better into common core automation library maybe?

Reply:
Done